### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by ukamila**
funnily enough contact page is not changed in this PR, but rather new projects were added and small css changes

### CODE REVIEW

File: .vscode/settings.json
Comment (gregfrasco) [inline .vscode/settings.json:1]: ignore .vscode folder

File: .gitignore
Comment (gregfrasco) [inline .gitignore:1]: remove white space changes

**Comment by gregfrasco** [comment]
Cannot find module 'react-awesome-reveal'

**Review by gregfrasco — CHANGES_REQUESTED** [review]
minor changes, but couldn't load the page to comment on that.

File: .vscode/settings.json
Comment (gregfrasco) [inline .vscode/settings.json:1]: Also we need to remove this file from being tracked

**Review by gregfrasco — APPROVED** [review]
Minor changes

**Review by gregfrasco — APPROVED** [review]

**Comment by ukamila** [comment]
everything's working now, but pending changes

**Pull request merged**